src/main/java/seedu/contax/ui/onboarding/OnboardingWindow.java

```diff
@@ -119,126 +117,270 @@ public void translate(double x, double y) {
+     * <br>- 3: cover all, showing only the person list
+     * @param option the overlay option
      */
-    public void processOverlayOption(int option) {
-        if (option == 0) {
+    private void processOverlayOption(int option) {
+        switch (option) {
+        case 0:
             coverAll();
-        } else if (option == 1) {
-            showOnly(menuBar);
-        } else if (option == 2) {
-            showOnly(commandBoxPlaceholder);
-        } else if (option == 3) {
-            showOnly(personList);
+            break;
+        case 1:
+            showOnly(menuBar, 0);
+            break;
+        case 2:
+            showOnly(commandBoxPlaceholder, 0);
+            break;
+        case 3:
+            showOnly(personList, 2);
+            break;
+        default:
+            break;
         }
     }
 
     /**
-     * Moves the InstructionLabel based on the given option
+     * Moves the InstructionLabel to a position based on the given option.
+     * <br><br>
+     * Options are as follows:
+     * <br>- 0: center of window
+     * <br>- 1: top right of menu bar
+     * <br>- 2: top right of command box
+     * <br>- 3: top right of result display
+     * <br>- 4: middle right of result display
      * @param option Position option
      */
-    public void processInstructionPosition(int option) {
-        if (option == 0) {
+    private void processInstructionPosition(int option) {
+        switch (option) {
+        case 0:
             instructionLabel.setCenter(stage.heightProperty(), stage.widthProperty());
-        } else if (option == 1) {
+            break;
+        case 1:
             instructionLabel.translate(menuBar.layoutXProperty(), menuBar.layoutYProperty());
-        } else if (option == 2) {
+            break;
+        case 2:
             instructionLabel.translate(commandBoxPlaceholder.layoutXProperty(),
                     commandBoxPlaceholder.layoutYProperty());
-        } else if (option == 3) {
+            break;
+        case 3:
             instructionLabel.translate(resultDisplayPlaceholder.layoutXProperty(),
                     resultDisplayPlaceholder.layoutYProperty());
-        } else if (option == 4) {
+            break;
+        case 4:
             instructionLabel.translate(
                     personList.layoutXProperty().add(0),
                     personList.layoutYProperty().add(
                             resultDisplayPlaceholder.heightProperty().multiply(1.5)
                     ));
+            break;
+        default:
+            break;
+        }
+    }
+
+    /**
+     * Processes an operation on this window based on the given operation id
+     * <br><br>
+     * Options are as follows:
+     * <br>- 0: close this window and open main window
+     * <br>- 1: format the given message
+     * <br>- 2: find the latest person
+     * <br>- 3: format command for enforcement of  user input
+     * <br>- 5: remove latest person
+     * <br>- 6: list all persons
```

Comment:
This function looks like its doing a lot of different things, consider perhaps using anonymous lambdas if you need specific actions for each story.

Reply:
Great suggestion. Updated accordingly